lineage/data/Lineage_collections.py

```diff
@@ -28,7 +23,8 @@ def popout_single_lineages(lineages):
 lap02 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00701_A5_1_V4.xlsx")]
 lap03 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00801_A5_1_V4.xlsx")]
 
-Lapatinib_Control = lap01 + lap012 + lap02 + lap03
+lapatinib_Control = lap01 + lap012 + lap02 + lap03
+Lapatinib_Control = fix_obs(lapatinib_Control)
```

Comment:
Why is this being done over and over after import? Shouldn't the fix be on import since it affects every file?

Reply:
fixed.